napari/_qt/widgets/qt_viewer_status_bar.py

```diff
@@ -30,6 +35,12 @@ def __init__(self, parent: '_QtMainWindow') -> None:
     def setHelpText(self, text: str) -> None:
         self._help.setText(text)
 
+    def setStatusText(self, text: str) -> None:
+        # print(dir(self.parent()))
```

Comment:
```suggestion
```

Reply:
oops I was meant to start a review, not add a single comment... sorry for the noise